src/lib.rs

```diff
@@ -2325,6 +2345,39 @@ impl Build {
         .map(|x| x.to_owned()))
     }
 
+    /// Some platforms have multiple, compatible, canonical prefixes. Look through
+    /// each possible prefix for a compiler that exists and return it. The prefixes
+    /// should be ordered from most-likely to least-likely.
+    fn find_working_gnu_prefix(&self, prefixes: &[&'static str]) -> Option<&'static str> {
+        let windows_host = self
+            .get_host()
+            .map(|host| host.contains("windows"))
+            .unwrap_or(false);
+        let suffix = if self.cpp { "-g++" } else { "-gcc" };
+        let extension = if windows_host { ".exe" } else { "" };
```

Comment:
I think this can be replaced with `std::env::consts::EXE_SUFFIX` perhaps?

Reply:
I had no idea this was a thing. Thanks for pointing it out.